pkg/cluster/executor/ssh.go

```diff
@@ -301,7 +301,17 @@ func (e *NativeSSHExecutor) Execute(cmd string, sudo bool, timeout ...time.Durat
 		defer cancel()
 	}
 
-	args := []string{"ssh", "-o", "StrictHostKeyChecking=no"}
+	ssh := "ssh"
+
+	isExec := func(mode os.FileMode) bool { return mode&0111 == 0111 }
```

Comment:
Maybe we could put this in `pkg/utils/ioutil.go` to make it reusable for other packages

Reply:
LGTM, rewrite it right now.